code/modules/atmospherics/machinery/atmosmachinery.dm

```diff
@@ -204,8 +223,9 @@
 	update_appearance()
 
 /obj/machinery/atmospherics/update_icon()
-	. = ..()
 	update_layer()
+	update_cap_visuals()
+	. = ..()
```

Comment:
```suggestion
	return ..()
```

means the same exact thing compiler-wise, but it's nicer to have it like this because we're `return`ing at the end of the proc.